jvb/rootfs/defaults/jvb.conf

```diff
@@ -137,6 +138,13 @@ jmt {
 ice4j {
     harvest {
         mapping {
+            aws {
+{{ if $DISABLE_AWS_HARVESTER -}}
```

Comment:
Since it's a boolean, and we have a default value, why not set it unconditionally?